app/main.py

```diff
@@ -1,34 +1,86 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(self,
+                 start: tuple,
+                 end: tuple,
+                 is_drowned: bool = False) -> None:
+        self.is_drowned = is_drowned
 
-    def get_deck(self, row, column):
-        # Find the corresponding deck in the list
-        pass
+        if start == end:
+            self.decks = [Deck(*start), ]
+        if start[0] == end[0]:
+            self.decks = [Deck(start[0], i)
+                          for i in range(start[1], end[1] + 1)]
+        if start[1] == end[1]:
+            self.decks = [Deck(i, start[1])
+                          for i in range(start[0], end[0] + 1)]
```

Comment:
Why do you need to use if statement instead just use for-loop

Reply:
I'm just overthinking this :)